javalib/src/main/scala/java/util/concurrent/locks/ReentrantLock.scala

```diff
@@ -0,0 +1,16 @@
+package java.util.concurrent.locks
+
+class ReentrantLock(private[this] val fair: Boolean) {
+
+  private var locked = false
+
+  def this() = this(false)
+
+  def lock(): Unit = locked = true
+
+  def unlock(): Unit = locked = false
+
+  def isLocked(): Boolean = locked
+
+  final def isFair: Boolean = fair
```

Comment:
In a Java class, methods without parameters need to have `()`. See https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#parameterless-methods